graphql2/graphqlapp/config.go

```diff
@@ -16,6 +16,7 @@ func (a *Query) IntegrationKeyTypes(ctx context.Context) ([]graphql2.Integration
 		{ID: "grafana", Name: "Grafana", Label: "Grafana Webhook URL", Enabled: true},
 		{ID: "site24x7", Name: "Site 24x7", Label: "Site24x7 Webhook URL", Enabled: true},
 		{ID: "prometheusAlertmanager", Label: "Alertmanager Webhook URL", Name: "Prometheus Alertmanager", Enabled: true},
+		{ID: "universal", Label: "Universal Integration Key", Name: "Universal Integration Key", Enabled: true},
```

Comment:
This one needs to be behind the flag, we don't want to allow users to create a key that doesn't work (unless the exp flag is set)